Formula/servefiles.rb

```diff
@@ -0,0 +1,35 @@
+class Servefiles < Formula
+  desc "Simple Python script for serving local files to FBI's remote installer."
+  homepage "https://gbatemp.net/threads/release-fbi-open-source-cia-installer.386433/"
+  url "https://github.com/Steveice10/FBI/archive/2.4.12.tar.gz"
+  sha256 "aadbc150fee385aa8c0d8ef2b1ab1deba3d5bbf3b65fd3497be565b1b2a380d9"
+  head "https://github.com/Steveice10/FBI.git"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    bin.install "servefiles/servefiles.py"
+    bin.install "servefiles/sendurls.py"
+    prefix.install "servefiles/README.md"
+    prefix.install "LICENSE.txt"
```

Comment:
This line isn't necessary since metafiles like `LICENSE` in the root directory are installed automatically.